### ISSUE TITLE
Problem validating Resources/FlowLogs/Properties/LogFormat

### ISSUE THREAD

**Issue by asantos-fuze** [aws-cloudformation/cfn-python-lint#1644]
*cfn-lint version: (`cfn-lint 0.34.1`)*

*Description of issue.*

I get the error `E1029 Found an embedded parameter "${version}" outside of an "Fn::Sub" at Resources/FlowLogs/Properties/LogFormat` when I try to validate a CFn template that has a custom Log Format

Please provide as much information as possible:
* Test Template:
```yaml
Resources:
  S3Bucket:
    Type: "AWS::S3::Bucket"

  FlowLogs:
    Type: "AWS::EC2::FlowLog"
    Properties:
      LogDestination: !GetAtt S3Bucket.Arn
      LogDestinationType: s3
      ResourceId: vpc-1234
      ResourceType: VPC
      TrafficType: ALL
      LogFormat: "${version} ${account-id} ${interface-id} ${srcaddr} ${dstaddr} ${srcport} ${dstport} ${protocol} ${packets} ${bytes} ${start} ${end} ${action} ${log-status} ${vpc-id} ${subnet-id} ${instance-id} ${tcp-flags} ${type} ${pkt-srcaddr} ${pkt-dstaddr} ${region} ${az-id}"
```
* Error During Validation:
```
cfn-lint -t test.yaml
sys:1: Warning: Python 2.7 has reached end of life. cfn-lint will end support for python 2.7 on December 31st, 2020.
E1029 Found an embedded parameter "${version}" outside of an "Fn::Sub" at Resources/FlowLogs/Properties/LogFormat
test.yaml:16:7

E1029 Found an embedded parameter "${srcaddr}" outside of an "Fn::Sub" at Resources/FlowLogs/Properties/LogFormat
test.yaml:16:7

E1029 Found an embedded parameter "${dstaddr}" outside of an "Fn::Sub" at Resources/FlowLogs/Properties/LogFormat
test.yaml:16:7

E1029 Found an embedded parameter "${srcport}" outside of an "Fn::Sub" at Resources/FlowLogs/Properties/LogFormat
test.yaml:16:7

E1029 Found an embedded parameter "${dstport}" outside of an "Fn::Sub" at Resources/FlowLogs/Properties/LogFormat
test.yaml:16:7

E1029 Found an embedded parameter "${protocol}" outside of an "Fn::Sub" at Resources/FlowLogs/Properties/LogFormat
test.yaml:16:7

E1029 Found an embedded parameter "${packets}" outside of an "Fn::Sub" at Resources/FlowLogs/Properties/LogFormat
test.yaml:16:7

E1029 Found an embedded parameter "${bytes}" outside of an "Fn::Sub" at Resources/FlowLogs/Properties/LogFormat
test.yaml:16:7

E1029 Found an embedded parameter "${start}" outside of an "Fn::Sub" at Resources/FlowLogs/Properties/LogFormat
test.yaml:16:7

E1029 Found an embedded parameter "${end}" outside of an "Fn::Sub" at Resources/FlowLogs/Properties/LogFormat
test.yaml:16:7

E1029 Found an embedded parameter "${action}" outside of an "Fn::Sub" at Resources/FlowLogs/Properties/LogFormat
test.yaml:16:7

E1029 Found an embedded parameter "${type}" outside of an "Fn::Sub" at Resources/FlowLogs/Properties/LogFormat
test.yaml:16:7

E1029 Found an embedded parameter "${region}" outside of an "Fn::Sub" at Resources/FlowLogs/Properties/LogFormat
test.yaml:16:7
```

I've run manually `cfn-lint -u` but the validation on cfn-lint keeps failing.
The template is deployed successfully to AWS via Cloudformation

**Comment by PatMyron** [comment]
Thanks for the report. Should be fixed for the next upcoming release

Can workaround with [resource-level ignores](https://github.com/aws-cloudformation/cfn-python-lint#resource-based-metadata) until then:

```yaml
  Resource:
    Type: AWS::EC2::FlowLog
    Metadata:
      cfn-lint:
        config:
          ignore_checks:
          - E1029
```
